grails-app/views/emails/_email_share_invite.gsp

```diff
@@ -1,7 +1,7 @@
 <p>Hello,<br></p>
 
 <p>${sharer} wants to share a canvas with you on the <a href="https://www.streamr.com/core/">Streamr Core</a> real-time data app.<br/>
```

Comment:
We should change the text "share a canvas" to something else, as this email gets sent also if the resource in question is not a canvas, but a stream or a dashboard.

`${sharer} wants to share with you`...

Reply:
@hpihkala I've changed it to resource. Any better? Ideas?